### ISSUE TITLE
Build fails when curl is missing

### ISSUE THREAD

**Issue by yoctozepto** [ipmitool/ipmitool#42]
When `curl` is missing, `lib/create_pen_list` creates an unusable include file with the following contents:

```
{ , "" },
```

and the build fails with a syntax error. The build should fail early when `curl` cannot be called and `configure` should check whether `curl` is available.

I can fix it and submit a PR but I am time-constrained at the very moment, hence posting an issue.

Have a good day.

**Comment by yoctozepto** [comment]
Additionally `make clean` does not remove generated `lib/ipmi_pen_list.inc.c`.

**Comment by yoctozepto** [comment]
Ideally, this should be moved to bootstrap anyway.

**Comment by AlexanderAmelkin** [comment]
This is a duplicate of #11.

**Comment by hdatma** [comment]
I have curl, and it fails anyway. 
In my case, the problem is conv. 

```
./create_pen_list test.txt                                                                                                                                                       
Generating IANA PEN list...
iconv: conversion from utf8 unsupported
iconv: try 'iconv -l' to get the list of supported encodings
-=O#-   #      #        #   
```
However, I have iconv and it is utf8 capable: 

```
>iconv --version | head -1
iconv (GNU libiconv 1.15)

>iconv -l | grep -i utf.*8
UTF-8
```
So the problem is at line 48 of create_pen_list, where "iconv -f utf8" should be "iconv -f UTF-8".

**Comment by AlexanderAmelkin** [comment]
@hdatma, it will all be addressed in context of #11.
